spinalcordtoolbox/deepseg/models.py

```diff
@@ -405,9 +405,10 @@
          'long_description': 'This segmentation model for spinal nerve rootlets segmentation uses a 3D U-Net '
                              'architecture, and was trained with the nnUNetV2 framework. It is a multiclass model, '
                              'outputting a single segmentation image containing 8 classes representing the C2-T1 '
-                             'dorsal and ventral spinal cord nerve rootlets. Training data consisted of 76 isotropic '
-                             'T2w and MP2RAGE images from healthy subjects from three datasets (spine-generic '
-                             'multi-subject, OpenNeuro ds004507, 7T MP2RAGE (private)).',
+                             'dorsal and ventral spinal cord nerve rootlets. Training data consisted of images from '
+                             'healthy subjects from three datasets: spine-generic multi-subject (3T T2w: n=21), '
```

Comment:
just fyi: I kept "subjects" (and not "participants" as suggested in [this comment](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4895#discussion_r2079936575)) as we use "subjects" for all other tasks

Reply:
Sounds good!